src/components/StatisticsTable/index.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect, useState } from 'react';
+import {
+  makeStyles,
+  Paper,
+  Table,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableBody,
+  TableRow,
+} from '@material-ui/core';
+import API from '../../utils/API';
+
+type TableColumn = {
+  id: string,
+  label: string,
+  minWidth: number,
+  align?: 'right' | 'left' | 'center' | 'justify' | 'inherit',
+  format?: (value: any) => string,
+}
+
+const useStyles = makeStyles({
+  root: {
+    width: '100%',
+    maxWidth: 440,
+    margin: 'auto',
+    marginTop: '60px',
+  },
+  container: {
+    maxHeight: 440,
+  },
+  head: {
+    fontWeight: 600,
+  }
+});
+
+export default function StatisticsTable(): JSX.Element {
+  const [rows, setRows] = useState([]);
+  const classes = useStyles();
+  const columns: TableColumn[] = [
+    { id: 'userName', label: 'Name', minWidth: 70 },
+    {
+      id: 'date',
+      label: 'Game Date',
+      minWidth: 100,
+      align: 'right',
+      format: (value: string) =>
+        new Date(value).toLocaleString('en-US'),
+    },
+    {
+      id: 'score',
+      label: 'Score',
+      minWidth: 70,
+      align: 'right',
+      format: (value: number) => value.toLocaleString('en-US'),
+    },
+  ];
+
+  useEffect(() => {
+    let isMounted = true;
+    const fetchStr = 'api/get_users';
+    API.get(fetchStr)
+      .then((res) => res.data)
+      .then((data) => {
+        if (!data) {
```

Comment:
Need to add error handling in requests.
You could use "catch" or "reject" scenarios, as you wish.